### PR TITLE
zh trans Create kubeadm_join_phase_preflight.md

### PR DESCRIPTION
**Pull request by jiazxjason**
kubeadm_join_phase_preflight.md

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_preflight.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+<!--
+
+Options
+
+-->
+
+选项
+
+    <tr>
+      <td colspan="2">--apiserver-advertise-address string</td>
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_preflight.md]:
```suggestion
      <td colspan="2">--apiserver-advertise-address 字符串</td>
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_preflight.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">If the node should host a new control plane instance, the IP address the API Server will advertise it's listening on. If not set the default network interface will be used.</td>
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_preflight.md]: we start here, and we go to line 114. Need to be translated

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_preflight.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+-->
+
+从父命令继承的选项
+
+    <tr>
+      <td colspan="2">--rootfs string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">[EXPERIMENTAL] The path to the 'real' host root filesystem.</td>
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_preflight.md]: And here you need

**Comment by markthink** [comment]
@jiazxjason 提交不完整，自检预览一下

https://deploy-preview-16474--k8s-v1-14.netlify.com/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_join_phase_preflight/

**Comment by tengqm** [comment]
the submitted file is not a markdown file.

**Pull request closed without merging**